iocore/eventsystem/I_VConnection.h

```diff
@@ -116,13 +116,16 @@
 //
 //////////////////////////////////////////////////////////////////////////////
 
-/** Used in VConnection::shutdown(). */
-enum ShutdownHowTo_t {
-  IO_SHUTDOWN_READ = 0,
-  IO_SHUTDOWN_WRITE,
-  IO_SHUTDOWN_READWRITE,
+enum class IOShutdown {
+  READ      = SHUT_RD,
+  WRITE     = SHUT_WR,
+  READWRITE = SHUT_RDWR,
 };
 
+constexpr IOShutdown IO_SHUTDOWN_READ      = IOShutdown::READ;
+constexpr IOShutdown IO_SHUTDOWN_WRITE     = IOShutdown::WRITE;
+constexpr IOShutdown IO_SHUTDOWN_READWRITE = IOShutdown::READWRITE;
```

Comment:
-1. We should be consistent with the existing code. If we are changing the style of IOCore we should do all of it, not just a single enum.